docs/events-api.asciidoc

```diff
@@ -90,20 +95,20 @@ An example error response might look something like this:
 <3> An immediately returning non-event related error
 <4> The number of accepted events
 
-If you're developing an agent, these errors can be useful for debugging your agent while building it. 
+If you're developing an agent, these errors can be useful for debugging. 
 
 [[events-api-schema-definition]]
 [float]
 === Event API Schemas
 
 The APM Server uses a collection of JSON Schemas for validating requests to the intake API:
 
-* <<metadata-api, Metadata>>
-* <<transaction-api, Transactions>>
-* <<span-api, Spans>>
-* <<error-api, Errors>>
-* <<metricset-api, Metrics>>
-* <<example-intake-events, Example Request Body>>
+* <<metadata-api>>
+* <<transaction-api>>
+* <<span-api>>
+* <<error-api>>
+* <<metricset-api>>
+* <<example-intake-events>>
```

Comment:
@simitt We don't have a section here for the new Jaeger endpoints. My gut instinct is that it isn't necessary, but I'm curious to hear your thoughts.

Reply:
I am with you, the Jaeger data are sent via a different protocol and endpoint and is not part of the events intake API.